helm/nginx-ingress-controller-app/templates/controller-deployment.yaml

```diff
@@ -27,7 +27,7 @@ spec:
         giantswarm.io/service-type: "managed"
         k8s-app: {{ .Values.controller.k8sAppLabel }}
       annotations:
-        releasetime: {{ $.Release.Time }}
+        releaseRevision: {{ $.Release.Revision | quote }}
```

Comment:
https://v2.helm.sh/docs/chart_template_guide/#built-in-objects Release.Revision is there for helm v2 so this should work there too.